.github/workflows/build.yml

```diff
@@ -18,17 +18,18 @@ jobs:
     tests:
         runs-on: ubuntu-latest
 
-        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, wkhtmltopdf: ${{ matrix.wkhtmltopdf }}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, wkhtmltopdf: ${{ matrix.wkhtmltopdf }}, State Machine Adapter ${{ matrix.state_machine_adapter }}"
 
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1"]
-                symfony: ["^5.4", "^6.0"]
-                sylius: ["~1.11.4", "~1.12.0"]
+                php: ["8.1", "8.2", "8.3"]
+                symfony: ["^5.4.21", "^6.4"]
+                sylius: ["~1.12.0", "~1.13.x-dev",]
                 node: ["16.x"]
```

Comment:
We should probably adjust the tested versions of Node.js only to the supported ones (18 and 20), especially that in Sylius 1.13 we supports only these two versions

Reply:
Since 1.12 also allows it, I'd be for leaving only 20.x